mocks/layouts.go

```diff
@@ -0,0 +1,37 @@
+package mocks
+
+import (
+	"context"
+
+	"github.com/influxdata/chronograf"
+)
+
+var _ chronograf.LayoutStore = &LayoutStore{}
+
+type LayoutStore struct {
+	AddF    func(ctx context.Context, layout chronograf.Layout) (chronograf.Layout, error)
+	AllF    func(ctx context.Context) ([]chronograf.Layout, error)
+	DeleteF func(ctx context.Context, layout chronograf.Layout) error
```

Comment:
interesting. i would think this would take an id? i realize this is out of scope for this PR, but just curious.